elyra/airflow/bootstrapper.py

```diff
@@ -74,10 +75,28 @@ def __init__(self, **kwargs: Any) -> None:
         # Infer secure from the endpoint's scheme.
         self.secure = self.cos_endpoint.scheme == 'https'
 
-        self.cos_client = minio.Minio(self.cos_endpoint.netloc,
-                                      access_key=os.getenv('AWS_ACCESS_KEY_ID'),
-                                      secret_key=os.getenv('AWS_SECRET_ACCESS_KEY'),
-                                      secure=self.secure)
+        # get minio credentials provider
+        if "cos-user" in self.input_params and "cos-password" in self.input_params:
```

Comment:
Are we still passing this? I believe we removed it from input_params for security issues (leaking these details) and only pass them as ENV VARS?

Reply:
This is for the unit tests, it's not used in production.

Here is the [`_get_operation_instance()`](https://github.com/elyra-ai/elyra/blob/67779c3917a4587bd8a9f2e589addda0fc27a64d/elyra/tests/kfp/test_bootstrapper.py#L155-L156) function which sets `cos-user` and `cos-password` used in unit tests.